src/System.Net.WebHeaderCollection/ref/System.Net.WebHeaderCollection.cs

```diff
@@ -88,34 +88,34 @@ public partial class WebHeaderCollection : System.Collections.Specialized.NameVa
     {
         public WebHeaderCollection() { }
         protected WebHeaderCollection(System.Runtime.Serialization.SerializationInfo serializationInfo, System.Runtime.Serialization.StreamingContext streamingContext) { }
-        public override string[] AllKeys { get { throw null; } }
+        public override string?[] AllKeys { get { throw null; } }
```

Comment:
In what situation will AllKeys contain null?

Reply:
The InnerCollection.AllKeys called inside getter returning string?[]
```csharp
public override string?[] AllKeys
{
    get
    {
        return InnerCollection.AllKeys;
     }
}
```

Reply:
Because the base collection lets you add nulls.  But is there a way to get a null into the derived class?

Reply:
Oh yeah, there will be no null name/key, will update thanks!